### ISSUE TITLE
Glossary term removed from tooltip

### ISSUE THREAD

**Issue by jstoeffler** [CodeAtCode/Glossary#30]
If the glossary term should be in the excerpt, it's removed from the tooltip.

So if I have:

Glossary term: Foobar
Glossary text: The terms foobar fubar, or foo, bar are sometimes used as placeholder names.
Glossary search terms: foobar

The tooltip will display "The terms fubar, or foo, bar are sometimes used as placeholder names."

It's funny, because when the tooltip fades out (because I leave the mouse), it appears for a short while (and it's linked).

So basically I guess there's a recursion.

**Comment by jstoeffler** [comment]
Just noticed: this doesn't happeny if there's an external link.

**Comment by Mte90** [comment]
I don't understand if is a CSS problem but with the case example is easy to replicate and study that problem.

**Comment by EugenioPetulla** [comment]
@jstoeffler  First strange UX issue I see is that you missed the fact that the title Foobar is not case sensistive so if you write Foobar in the title it also works as foobar. Try to leave the search terms input blank because now is like your are passing the term twice to the search engine. :)

I recreate the problem on my demo platform.

@Mte90 we have to find a way to communicate the "case sensitive" issue, maybe using a tooltip for the search terms input.

**Comment by jstoeffler** [comment]
Oh, interesting. 
So the title is already included into the list of search terms ?

I guess it makes sense, but it definitely wasn't clear to me. I also figured out later that plurals are also taken into account.

So if I understand the "Related search terms" field could (should?) be called "Additional search terms", right? 
And there could be a little text saying something like "plurals and lowercase versions are already taken into account", right?

**Comment by Mte90** [comment]
You're right, we have to improve that label but yes they are additional.

**Comment by Mte90** [comment]
I checked the code, the terms are case insensitive but there is no support for plurals.

**Comment by EugenioPetulla** [comment]
No plurals, just case insensitive, I confirm it. Good job guys! :D
